### PR TITLE
Chore: remove @google-cloud dependency

### PR DESCRIPTION
**Pull request by rodrigo4244**
Because we don't use Google Cloud anymore and this lib was raising `nsp` errors on other projects.

### CODE REVIEW

**Comment by rodrigo4244** [comment]
With this patch nsp pass on Emissary:
image

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@invisible/logger",
-  "version": "0.0.13",
+  "version": "0.0.14",
```
Comment (kewitz) [inline package.json]: This is a breaking change, let's bump this to `1.0.0` and start a changelog.

**Comment by rodrigo4244** [comment]
I don't have access to the `CHANGELOG_WEBHOOK_URL`. And I can't import from other projects too :(

**Comment by kewitz** [comment]
@rodrigo4244 smash similar commits and merge it.

**Pull request merged**